### PR TITLE
fix(ui): sample app version bump

### PR DESCRIPTION
**Pull request by sachaarbonel**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: packages/stream_chat_flutter/example/pubspec.yaml [hunk 1]
```diff
@@ -27,9 +27,9 @@ dependencies:
   cupertino_icons: ^1.0.3
   flutter:
     sdk: flutter
-  stream_chat_flutter: ^2.2.1
-  stream_chat_localizations: ^1.1.0
-  stream_chat_persistence: ^2.2.0
+  stream_chat_flutter: ^3.4.0
+  stream_chat_localizations: ^2.1.0
+  stream_chat_persistence: ^3.1.0
```
Comment (imtoori) [inline packages/stream_chat_flutter/example/pubspec.yaml:32]: @sachaarbonel isn't it better to use the local version from path?

**Review by imtoori — CHANGES_REQUESTED** [review]

File: packages/stream_chat_flutter/example/pubspec.yaml (hunk 1 above)
Comment (xsahil03x) [reply packages/stream_chat_flutter/example/pubspec.yaml:32]: isn't really necessary as we run them after `melos bs` which does the same thing.

Comment (imtoori) [reply packages/stream_chat_flutter/example/pubspec.yaml:32]: yeah but then once we push a new release we need to update this as well and it's very easy to forget

Comment (xsahil03x) [reply packages/stream_chat_flutter/example/pubspec.yaml:32]: fine then 👍🏼

Comment (imtoori) [reply packages/stream_chat_flutter/example/pubspec.yaml:32]: @sachaarbonel can you update this?

**Pull request merged**